target/linux/ipq806x/files-4.9/arch/arm/boot/dts/qcom-ipq8064-ea8500.dts

```diff
@@ -404,3 +414,7 @@
 		};
 	};
 };
+
+/delete-node/ &pcie2_pins;
+/delete-node/ &pcie2;
+ 
```

Comment:
Why do you need to delete the node? Can't you just remove the `status = "ok"` part? This should cause the device to never be brought up, and the pins should also never get referenced that way.
Since this changeset has been sitting there for a while now, I'm fine with merging it as-is now and cleaning it up later.